react/TabbedShelf.js

```diff
@@ -0,0 +1,3 @@
+import TabbedShelf from './TabbedShelf/index'
```

Comment:
I think the shelf is the only app that doesn't follow our folder structure. 

```
react/
    components/
```
Can you do this?

Reply:
done